src/app/core/services/github.service.ts

```diff
@@ -34,6 +34,8 @@ let ORG_NAME = '';
 let MOD_ORG = '';
 let REPO = '';
 let DATA_REPO = '';
+const MAX_SIZE_PER_PAGE = 100;
```

Comment:
```suggestion
const NUMBER_OF_LABELS_PER_PAGE = 100;
```

Reply:
For this, I named it MAX_SIZE_PER_PAGE as the maximum results per page is 100 for many of the other paginated APIs too (E.g. List assignees, issue comments). I was thinking a general name like this might be better as it would allow us to reuse it in other parts of the service (fetching of issues, etc).

What do you think about a more general name like `MAX_RESULTS_PER_PAGE` or should we still go with your suggestion `NUMBER_OF_LABELS_PER_PAGE`?

Reply:
I see, then we can go with `MAX_ITEMS_PER_PAGE` ba. 

Size is too vague of a term also...  😅